src/garage/tf/q_functions/continuous_cnn_q_function.py

```diff
@@ -184,11 +191,10 @@ def get_qval(self, observation, action):
                 corresponding to each (obs, act) pair.
 
         """
-        if isinstance(self._env_spec.observation_space, akro.Image):
-            if len(observation.shape) <= 3:
-                observation = self._env_spec.observation_space.unflatten(
-                    observation)
-            observation = observation / 255.0
+        if len(observation[0].shape) < len(self._obs_dim):
```

Comment:
What does this line do?

Reply:
Unflattens observations that have been flattened (`observation` has a batch dimension). Ideally this should be `get_qvals(observations, actions)` to avoid this confusion, though I left it as is to remain consistent with the other primitives.

Reply:
Sure I'll run DQN.